React/Views/UIView+React.h

```diff
@@ -83,9 +83,9 @@
 
 /**
  * This method attaches the specified controller as a child of the
- * the owning view controller of this view. Returns NO if no view
- * controller is found (which may happen if the view is not currently
- * attached to the view hierarchy).
+ * the owning view controller of this view. If host view controller not found,
```

Comment:
Hi 👋 ! Perhaps this should be phrased as "If a host view controller is not found..." or "If no host view controlled is found..."